docs/api/scheduler.md

```diff
@@ -1361,6 +1361,60 @@ Node details include host and rack name, capacity, resources, utilization, and a
 }
 ```
 
+## Node utilization
+
+Show how every node is distributed with regard to utilization.
+
```

Comment:
Sorry for late review, but I think we should emphasize the utilization type is 'dominant' resource type only.

Some thing like: 
"Show how every node is distributed with regard to dominant resource utilization."
(Dominant resource is defined as the highest utilised resource type on the root queue based on the registered resources.)

https://github.com/apache/yunikorn-core/blob/master/pkg/webservice/handlers.go#L353-L356

Reply:
Thanks for your review, updated the commit.